src/sql/src/session/vars.rs

```diff
@@ -909,11 +910,14 @@ const DATAFLOW_MAX_INFLIGHT_BYTES: ServerVar<usize> = ServerVar {
 /// The maximum number of in-flight bytes emitted by persist_sources feeding _storage
 /// dataflows_. This is distinct from `DATAFLOW_MAX_INFLIGHT_BYTES`, as this will
 /// supports more granular control (within a single timestamp).
+/// Currently defaults to 256MiB = 268435456 bytes
+/// Note: Backpressure will only be turned on if disk is enabled based on
+/// `storage_dataflow_max_inflight_bytes_disk_only` flag
 const STORAGE_DATAFLOW_MAX_INFLIGHT_BYTES: ServerVar<Option<usize>> = ServerVar {
     name: UncasedStr::new("storage_dataflow_max_inflight_bytes"),
-    value: &None,
+    value: &Some(268435456),
```

Comment:
can we make this the math?
```suggestion
    value: &Some(256 *1024 * 1024),
```